azure-pipelines.yml

```diff
@@ -0,0 +1,486 @@
+        RunArguments: docker run --privileged --rm -e ROOTFS_DIR=/crossrootfs/arm
+          --name ubuntu-14.04-cross-e435274-20180323032140
+          -v "$(SourcesDirectory):/root/coresetup"
+          -w="/root/coresetup"
+          -e "PACKAGEVERSIONPROPSURL=$(PackageVersionPropsUrl)"
+          microsoft/dotnet-buildtools-prereqs:ubuntu-14.04-cross-e435274-20180323032140
+        BuildArguments: -OfficialBuildId=$(OfficialBuildId)
+          -ConfigurationGroup=$(_BuildConfiguration)
+          -PortableBuild=true
+          -strip-symbols
+          -SkipTests=true
+          -CrossBuild=true
+          -TargetArchitecture=arm
+          -- /p:StabilizePackageVersion=$(IsStable)
+        PublishArguments: /p:PublishType=$(PublishType)
+          /p:TargetArchitecture=arm
+          /p:ConfigurationGroup=$(_BuildConfiguration)
+          /p:PortableBuild=true
+          /p:OSGroup=Linux
+          /p:StabilizePackageVersion=$(IsStable)
+        _TargetArchitecture: arm
+      pool:
+        ${{ if eq(variables['System.TeamProject'], 'public') }}:
+          name: dnceng-linux-external-temp
+        ${{ if ne(variables['System.TeamProject'], 'public') }}:
+          name: dnceng-linux-internal-temp
+      strategy:
+        maxParallel: 2
+        matrix: 
+          debug:
+            _BuildConfiguration: Debug
+          release:
+            _BuildConfiguration: Release
+
+    # Linux Alpine 3.6
+    - job:
+      agentOS: Linux
+      displayName: Linux_x64_Alpine36
+      variables:
+        RunArguments: docker run --privileged --rm
+          --name alpine-3.6-3148f11-20171119021156
+          -v "$(SourcesDirectory):/root/coresetup"
+          -w="/root/coresetup"
+          -e "PACKAGEVERSIONPROPSURL=$(PackageVersionPropsUrl)"
+          microsoft/dotnet-buildtools-prereqs:alpine-3.6-3148f11-20171119021156
+        BuildArguments: -OfficialBuildId=$(OfficialBuildId)
+          -ConfigurationGroup=$(_BuildConfiguration)
+          -TargetArchitecture=x64
+          -PortableBuild=false
+          -strip-symbols
+          -SkipTests=$(SkipTests)
+          -- /p:StabilizePackageVersion=$(IsStable)
+          /p:OutputRid=linux-musl-x64
+        PublishArguments: /p:PublishType=$(PublishType)
+          /p:TargetArchitecture=x64
+          /p:ConfigurationGroup=$(_BuildConfiguration)
+          /p:PortableBuild=false
+          /p:OSGroup=Linux
+          /p:OutputRid=linux-musl-x64
+          /p:StabilizePackageVersion=$(IsStable)
+        _TargetArchitecture: x64
+      pool:
+        ${{ if eq(variables['System.TeamProject'], 'public') }}:
+          name: dnceng-linux-external-temp
+        ${{ if ne(variables['System.TeamProject'], 'public') }}:
+          name: dnceng-linux-internal-temp
+      strategy:
+        maxParallel: 2
+        matrix: 
+          debug:
+            _BuildConfiguration: Debug
+          release:
+            _BuildConfiguration: Release
+
+    # Linux Rhel6
+    - job:
+      agentOS: Linux
```

Comment:
what are these agentOS variables used for?

Reply:
This can be removed, will take care of this.